core/templates/dev/head/pages/collection_editor/collection_editor_navbar_directive.html

```diff
@@ -30,22 +30,18 @@
     </li>
 
     <!-- History -->
-    {% if SHOW_COLLECTION_NAVIGATION_TAB_HISTORY %}
-    <li ng-class="{'active': getTabStatuses().active === 'history'}">
+    <li ng-class="{'active': getTabStatuses().active === 'history'}" ng-if="SHOW_COLLECTION_NAVIGATION_TAB_HISTORY">
```

Comment:
if get it correctly you mean i should get rid of those two `li` elements for history and stats tabs right??